README.md

```diff
@@ -1163,6 +1178,18 @@ You can find our Code of Conduct [here](https://github.com/CSwala/awesome-comput
 
 This project follows the [MIT License](https://choosealicense.com/licenses/mit/).
 
+## Contributors 🥰:star:
```

Comment:
This is already there , why adding it again?

Reply:
I am not adding again. That issue was also solved by me so whenever i create  any pull request in a particular Project all the changes done by me is being shown

Reply:
Then after that PR was merged , you have not fetched the develop branch
Fetch the develop , then this should not happen

Reply:
ok, I got it

Reply:
@riturajjain2000  actually i am new with git so unable to do fetching can you help?